docs/guide.md

```diff
@@ -54,6 +54,26 @@ If an access token is not provided, the following rate limits apply:
 - 25 container builds per day
 - 250 container pulls per hour
 
+## Reproducible builds with SHA256 container image references
+
+Referring to the SHA256 hash of a container imagine is a common best practice in pipeline development. For example, the SHA256 hash for a container might be similar to `sha256:3235...ce8f`. If you use the Wave service to augment your containers, such as to add Fusion to a container, the original SHA256 hash of the image is no longer correct.
+
+To accommodate the new layers added to a container image augmented with Wave, you can use the freeze feature of Wave. In your pipeline configuration, ensure that you specify the following settings:
```

Comment:
```suggestion
## Known limitation 

### Use of sha256 digest in the image name

The use of sha256 digest in the image name, e.g. `ubuntu@sha256:3235...ce8f`, is not supported by the Wave augmentation process. This adding new layers to the image will result into a different sha256 digest id for the container image returned by Wave.   

A possible workaround consists of enabling the *free* feature, which causes Wave to rebuild the container with the required new layers. To enable this capability add in your pipeline configuration the following settings:
```

I've changed a bit. Possible it can be improved

Reply:
updated the suggestion a bit

Reply:
Nice - some tweaks on top of this suggested change:

```suggestion
## Known limitations

### Use of sha256 digest in the image name

Wave augmentation does not support the use of sha256 digests in image names, e.g., `ubuntu@sha256:3235...ce8f`. Adding new layers to the image will result in a different sha256 digest ID for the container image returned by Wave. The docker client will detect the hash mismatch and exit with an error.

A workaround is possible by enabling the Wave _freeze_ feature. This instructs Wave to rebuild the container with the additional augmentation layers, after which it is pushed to a docker registry of your choice. This new image can then be used in your pipeline with a fixed sha256 digest and will not need further augmentation by Wave.

To enable this capability, add the following settings to your pipeline configuration:
```